pkg/message/types.go

```diff
@@ -381,3 +384,28 @@ type Flowspec struct {
 	SpecHash       string              `json:"spec_hash,omitempty"`
 	Spec           []flowspec.Spec     `json:"spec,omitempty"`
 }
+
+// Stats defines a message format sent to as a result of BMP Stats Message
+type Stats struct {
+	Key                        string `json:"_key,omitempty"`
+	ID                         string `json:"_id,omitempty"`
+	Rev                        string `json:"_rev,omitempty"`
+	Sequence                   int    `json:"sequence,omitempty"`
+	RouterHash                 string `json:"router_hash,omitempty"`
+	RouterIP                   string `json:"router_ip,omitempty"`
+	RemoteBGPID                string `json:"remote_bgp_id,omitempty"`
+	RemoteASN                  uint32 `json:"remote_asn,omitempty"`
+	RemoteIP                   string `json:"remote_ip,omitempty"`
+	PeerRD                     string `json:"peer_rd,omitempty"`
+	Timestamp                  string `json:"timestamp,omitempty"`
+	DublicatePrefixs           uint32 `json:"dubprefixes,omitempty"`
```

Comment:
typo in the variable name, please use `duplicate_prefix` in json, it is easier to read